packages/core/docs/commercetools/composables/use-forgot-password.md

```diff
@@ -0,0 +1,184 @@
+
+```ts
+interface UseForgotPasswordErrors {
+  request: Error;
+  setNew: Error;
+}
+```
+
+## Getters
+
+- `getResetPasswordToken` - returns generated reset password token.
+
+- `isPasswordChanged` - returns a boolean value of a password set status.
+
+```ts
+interface ForgotPasswordGetters<ForgotPasswordResult> {
+  getResetPasswordToken: (result: ForgotPasswordResult) => string
+  isPasswordChanged: (result: ForgotPasswordResult) => boolean
+}
+
+interface ForgotPasswordResult {
+  resetPasswordResult: CreatePasswordResetTokenResponse;
+  setNewPasswordResult: ResetPasswordResponse;
+}
+
+type CreatePasswordResetTokenResponse = QueryResponse<'customerCreatePasswordResetToken', CustomerPasswordToken>;
+
+type ResetPasswordResponse = QueryResponse<'customerResetPassword', Customer>;
+
+type CustomerPasswordToken = {
+  customerId: Scalars["String"];
+  expiresAt: Scalars["DateTime"];
+  value: Scalars["String"];
+  id: Scalars["String"];
+  version: Scalars["Long"];
+  createdAt: Scalars["DateTime"];
+  lastModifiedAt: Scalars["DateTime"];
+  createdBy?: Maybe<Initiator>;
+  lastModifiedBy?: Maybe<Initiator>;
+}
+
+type Customer = Versioned & {
+  __typename?: "Customer";
+  customerNumber?: Maybe<Scalars["String"]>;
+  email: Scalars["String"];
+  password: Scalars["String"];
+  addresses: Array<Address>;
+  defaultShippingAddressId?: Maybe<Scalars["String"]>;
+  defaultBillingAddressId?: Maybe<Scalars["String"]>;
+  shippingAddressIds: Array<Scalars["String"]>;
+  billingAddressIds: Array<Scalars["String"]>;
+  isEmailVerified: Scalars["Boolean"];
+  customerGroupRef?: Maybe<Reference>;
+  externalId?: Maybe<Scalars["String"]>;
+  key?: Maybe<Scalars["String"]>;
+  firstName?: Maybe<Scalars["String"]>;
+  lastName?: Maybe<Scalars["String"]>;
+  middleName?: Maybe<Scalars["String"]>;
+  title?: Maybe<Scalars["String"]>;
+  locale?: Maybe<Scalars["Locale"]>;
+  salutation?: Maybe<Scalars["String"]>;
+  dateOfBirth?: Maybe<Scalars["Date"]>;
+  companyName?: Maybe<Scalars["String"]>;
+  vatId?: Maybe<Scalars["String"]>;
+  customerGroup?: Maybe<CustomerGroup>;
+  defaultShippingAddress?: Maybe<Address>;
+  defaultBillingAddress?: Maybe<Address>;
+  shippingAddresses: Array<Address>;
+  billingAddresses: Array<Address>;
+  storesRef: Array<KeyReference>;
+  stores: Array<Store>;
+  /** This field contains non-typed data. Consider using `customFields` as a typed alternative. */
+  customFieldsRaw?: Maybe<Array<RawCustomField>>;
+  /** This field would contain type data */
+  customFields?: Maybe<Type>;
+  custom?: Maybe<CustomFieldsType>;
+  id: Scalars["String"];
+  version: Scalars["Long"];
+  createdAt: Scalars["DateTime"];
+  lastModifiedAt: Scalars["DateTime"];
+  createdBy?: Maybe<Initiator>;
+  lastModifiedBy?: Maybe<Initiator>;
+  /** Custom fields are returned as a list instead of an object structure. */
+  customFieldList?: Maybe<Array<CustomField>>;
+};
```

Comment:
Link to `commercetools/api-client-reference.html#customer` instead of duplicating the whole interface